fang.go

```diff
@@ -19,6 +20,9 @@ import (
 const shaLen = 7
 
 // ErrorHandler handles an error, printing them to the given [io.Writer].
+//
+// Note that this will only be used if the STDERR is a terminal, and should
+// be used for styling only.
 type ErrorHandler = func(w io.Writer, styles Styles, err error)
```

Comment:
Two remarks

- Is there a need for this to be exported?

- you are not using it in your PR, right? Except in a comment I don't see it.


Why having it then?